src/redfish_collector/core/rawCollector.py

```
@@ -96,9 +96,17 @@ async def fetch_all(urls: list, token, serverAddress):
         logging.error("[%s] Fetch all URL error: %s", serverAddress, e)
         raise
 
-async def rawDataCollector(serverAddress,schemaContent,keyDict: dict,token,logLevel):
-    logFormat = '%(asctime)s [%(levelname)s] %(message)s'
-    logging.basicConfig(format=logFormat, level=logLevel.upper())
+async def rawDataCollector(serverAddress,schemaContent,keyDict: dict,token,logLevel,session,semaphore):
```

Ruff (0.15.17)

[warning] Too many return statements (7 > 6)

(PLR0911)